src/jwt/rsa-verifier.js

```diff
@@ -0,0 +1,76 @@
+/*
+Based on the work of Tom Wu
+http://www-cs-students.stanford.edu/~tjw/jsbn/
+http://www-cs-students.stanford.edu/~tjw/jsbn/LICENSE
+*/
+
+import {BigInteger} from 'jsbn';
+import SHA256 from 'crypto-js/sha256';
+
+var DigestInfoHead = {
+  sha1: '3021300906052b0e03021a05000414',
+  sha224: '302d300d06096086480165030402040500041c',
+  sha256: '3031300d060960864801650304020105000420',
+  sha384: '3041300d060960864801650304020205000430',
+  sha512: '3051300d060960864801650304020305000440',
```

Comment:
Why include entries for algorithms we do not support?

Reply:
I'll remove the ones we don't support. But I think it's worth keeping the supporting structure around (the 2 arrays)